examples/search-replace-create-setters/.expected/exec.sh

```diff
@@ -1,5 +1,5 @@
 #! /bin/bash
 
 # shellcheck disable=SC2016
-kpt fn eval --image gcr.io/kpt-fn/search-replace:unstable -- \
+kpt fn eval --image gcr.io/kpt-fn/search-replace:unstable --image-pull-policy never -- \
```

Comment:
I believe you needed this during the development only, right ?

Reply:
Not only during the development, but also in the e2e tests. When in e2e tests, there may be changes in a newly built container, we want to ensure the local image instead of the remote.

Let me try to convert the `exec.sh` to use the `args` which is supported by the test harness. I need to ensure `source="$(cat set-replicas.star)"` still works.

Reply:
Tried to use `args` supported by the test harness.
But it doesn't work due to some issue regarding with `"`.
I'd suggest to merge it to unblock Spotify and address this test harness issue later.